src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/model/GitLabRepository.java

```diff
@@ -0,0 +1,49 @@
+package de.tum.in.www1.artemis.service.connectors.gitlab.model;
+
+import java.net.URL;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GitLabRepository {
```

Comment:
append Dto to the class name

Reply:
rename folder from model into dto

Reply:
I renamed the GitLab and Bamboo `model` packages to `dto` and also refactored all containing class names